homeassistant/components/notify/lametric.py

```diff
@@ -78,6 +83,14 @@ def send_message(self, message="", **kwargs):
                 except AssertionError:
                     _LOGGER.error("Sound ID %s unknown, ignoring",
                                   data["sound"])
+            if "cycles" in data:
+                cycles = data['cycles']
+            if "priority" in data:
+                if data['priority'] in ["info", "warning", "critical"]:
+                    priority = data['priority']
+                else:
+                    _LOGGER.warning("Priority '%s' invalid, using default "
+                                    "'%s'" % (data['priority'], priority))
```

Comment:
Don't put additional quotes around the priority variables:
https://developers.home-assistant.io/docs/en/development_guidelines.html#log-messages

Pass the priority variables as parameters:
```py
_LOGGER.warning("Priority %s blabla %s", data['priority'], priority)
```